examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -2,9 +2,8 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import { KeyValueDataObject } from "@fluid-experimental/data-objects";
 import TinyliciousClient from "@fluid-experimental/tinylicious-client";
```

Comment:
Remind me - is this going to stay an independent package or be rolled into fluid-framework?

Reply:
Yeah, tinyliciousclient is going to change, and the static/root class is going to get pulled into the meta package. We're not quite there yet.

Reply:
`tinylicious-client` will remain an independent package and not be part of the uber package.

Reply:
But `tinylicious-client` would have a peer dependency on the uber package version?

Reply:
peer yes